setup.py

```diff
@@ -49,6 +49,13 @@ def read(*rnames):
       package_dir={'': 'src'},
       install_requires=[
           'setuptools',
+
+      ],
+      setup_requires=[
+          'pytest-runner',
```

Comment:
A, ok, I think that is a common misunderstanding of setuptools and dependency declaration in Python Eggs. 
http://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-dependencies
describe that only install_requires and extra_requires are used, if declared by external packages or pip install. Also only those are reflected in the *.egg-info/requires.txt file. 
setup_requires is only evaluated if 

    python setup.py install

or 

    python setup.py develop 

is used (--> see http://setuptools.readthedocs.io/en/latest/setuptools.html#new-and-changed-setup-keywords setup_requires).